src/main/proto/th2_grpc_common/common.proto

```diff
@@ -83,6 +83,17 @@ message RawMessage {
   bytes body = 2;
 }
 
+message GroupMessage {
+  oneof kind {
+    Message message = 1;
+    RawMessage raw_message = 2;
+  }
+}
+
+message MessageGroup {
+  repeated GroupMessage messages = 1;
+}
```

Comment:
@OptimumOpium what about this message? Should we rename it too?

Reply:
@ivandruzhinin I think that the name `MessageGroup` is good enough.
But maybe we should make it Message**s**Group because it has more than one message. On the other hand, the group will be converted to a single raw message at the end (and it is created from a single raw message we receive). So, we can leave it as it is, I guess

Reply:
`MessageGroup` seems to be a common name for an entity which holds a group of messages. For example: https://docs.spring.io/spring-integration/docs/current/api/org/springframework/integration/store/MessageGroup.html

Reply:
I agree. `MessageGroup` is more applicable here